test/check-application

```diff
@@ -88,6 +88,28 @@ class TestApplication(testlib.MachineCase):
         else:
             return self.admin_s.execute(cmd)
 
+    def login(self, system=True):
+        # HACK... explain
+
+        b = self.browser
+
+        self.login_and_go("/podman", superuser=system)
+        b.wait_visible("#app")
+
+        with self.browser.wait_timeout(30):
+            try:
+                b.wait_visible("#containers-containers")
+                b.wait_not_in_text("#containers-containers", "Loading")
+                b.wait_not_present("#overview div.pf-c-alert")
+            except testlib.Error:
+                if system:
+                    b.wait_in_text("#overview div.pf-c-alert .pf-c-alert__title", "User Podman service is also available")
+                    b.click("#overview div.pf-c-alert .pf-c-alert__action > button:contains(Start)")
+                    b.wait_not_present("#overview div.pf-c-alert")
```

Comment:
Ugh, that is really evil.. But I don't have a better idea either. :cry:

Reply:
yeah, I don't like it either, but I wasted way too much time with reporting to podman and it always just gets forgotten and then closed with "cannot reproduce anymore"